sections/semantics-document-metadata.include

```diff
@@ -348,6 +348,7 @@
     <dd><{link/rev}> — <a>Reverse link</a> relationship of the destination resource to this document (or subsection/topic)</dd>
     <dd><code>media</code> — Applicable media</dd>
     <dd><code>hreflang</code> — Language of the linked resource</dd>
+    <dd><code>integrity</code> — Metadata used in Subresource Integrity checks</dd>
```

Comment:
plese make this link so people can follow their nose and find out what that means.